packages/clerk-js/src/ui/common/EmailLinkVerify.tsx

```diff
@@ -1,12 +1,11 @@
 import { EmailLinkErrorCode, isEmailLinkError } from '@clerk/shared/error';
 import React from 'react';
 
-import type { VerificationStatus } from '../../utils/getClerkQueryParam';
+import { completeSignUpFlow, type VerificationStatus } from '../../utils';
```

Comment:
🙃 use a separate import for types or revert it to the previous state. This should probably fail in linting since we have `separate-type-imports` rule enabled. Eg
```typescript
import type { VerificationStatus } from '../../utils/';
import { completeSignUpFlow } from '../../utils';
```

Reply:
Ah, and I thought about doing it this way, but nothing failed this way. Thanks for letting me know!